wp-content/themes/hotelmaster-v2-01/stylesheet/style-custom.css

```diff
@@ -1,18 +1,15 @@
 body{ background: url('http://themes.goodlayers2.com/hotelmaster/wp-content/uploads/2015/03/bg-pattern.png'); }
 html.ltie9 body, body{ min-width: 1140px; } .container{ max-width: 1140px; } .gdlr-caption-wrapper .gdlr-caption-inner{ max-width: 1140px; }
-/*.body-wrapper.gdlr-boxed-style{
-	!*max-width: 1220px;*!
-} */
-.body-wrapper.gdlr-boxed-style .gdlr-header-wrapper{ max-width: 1220px; margin: 0px auto; }
+.body-wrapper.gdlr-boxed-style{ max-width: 1220px; } .body-wrapper.gdlr-boxed-style .gdlr-header-wrapper{ max-width: 1220px; margin: 0px auto; }
```

Comment:
Why did you remove this?.  I have commented this 'cause with this the websites responsiveness get lost in the mobile device.